apps/consent/env.ts

```diff
@@ -8,10 +8,12 @@ export const env = createEnv({
   },
   shared: {
     GRAPHQL_ENDPOINT: z.string().default("http://localhost:4002/graphql"),
+    NODE_ENV: z.string().default("development"),
```

Comment:
if we have a default to development then the likelihood we stumble upon the issue that it will be set as dev mode in production is quite high. 

I don't think we should have default here.